initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -445,6 +435,30 @@ startCmd() {
       fi
     fi
 
+    limits=(
+      MIN_Hn_nofile="32768"
+      MIN_Hc_core="unlimited"
+      MIN_Hu_nproc="8192"
+      MIN_Hr_rtprio="4"
+      MIN_Hl_memlock="unlimited"
+      MIN_Hs_stack="512" )
```

Comment:
I dont think stack of 512 is good for all.  I think its larger than this for most other things.